### ISSUE TITLE
Successfully installed from Docker, Now what?

### ISSUE THREAD

**Issue by asharamseervi** [esl/mongooseim-docker#15]
I now able to manage successully installed mongooseim from Docker.
```
===> Starting relx build process ...
===> Resolving OTP Applications from directories:
          /tmp/mongooseim/_build/prod/lib
          /usr/lib/erlang/lib
===> Resolved mongooseim-2.1.1
===> Including Erts from /usr/lib/erlang
===> release successfully created!
[2018-02-06 12:13:28] /builds/mongooseim-MongooseIM-master-2018-02-06_121317.tar.gz is ready
```

But, it's neither accessible to browser with floatingip:5222 or neither any commands are working. Kindly help, commands are like these:
```
root@mongooseim:/# mongooseimctl status
mongooseimctl: command not found
root@mongooseim:/# mongooseimctl start
mongooseimctl: command not found
```
and Docker PS commands gives:

```
root@mongooseim:/var/lib/docker/volumes# docker ps
CONTAINER ID        IMAGE                           COMMAND             CREATED             STATUS              PORTS               NAMES
bb97e90e96fb        mongooseim/mongooseim-builder   "/sbin/my_init"     3 hours ago         Up 3 hours                              mongooseim-builder
```

and When I try to start docker: I just stuck something like this:

```
root@mongooseim:/var/lib/docker# docker run -it mongooseim/mongooseim-builder
*** Running /etc/my_init.d/00_regen_ssh_host_keys.sh...
*** Running /etc/rc.local...
*** Booting runit daemon...
*** Runit started as PID 8
```

**Comment by michalwski** [comment]
I hope the getting started is briefly described in [README.md](https://github.com/esl/mongooseim-docker/blob/master/README.md). More information you can find in http://mongooseim.readthedocs.io/en/latest/user-guide/Getting-started/
MongooseIM is an XMPP server, in order to connect to it you need a xmpp client.

**Comment by asharamseervi** [comment]
Can anyone suggest or create tutorial? I'm desperately looking to try mongooseim for my upcoming project instead of openfire. I almost tried everything but unfortunately, something I'm missing, it's really frustrating when I stumbled on mongooseim since last September and another september headed.

But, not a single tutorial on youtube/ blog post where we begineers hit commands one-by-one to get successfully running mongooseim. Docs are not enough, they're for advanced users, what about intermediate or beginners?

Unfortunately, I tried both ejabberd and mongooseim, none of them worked, and none of them have any beginner tutorial.
